Dockerfile

```diff
@@ -0,0 +1,58 @@
+FROM node:buster AS builder
+
+# Create webapp directory
+RUN mkdir -p /app/webapp
+
+# Copy JSX source files
+COPY webapp/src /app/webapp/src
+COPY webapp/public /app/webapp/public
+COPY webapp/package*.json /app/webapp/
+
+WORKDIR /app/webapp/
+
+# Build up the frontend
+ARG REACT_APP_ENDPOINT
+ENV REACT_APP_ENDPOINT=$REACT_APP_ENDPOINT
+
+RUN npm install --only=prod
+RUN npm run build
+RUN mkdir -p /app/static/
+
+RUN cp -r build/* /app/static
+RUN mv /app/static/static/* /app/static/
+RUN rm -r /app/static/static
+
+
+FROM python:buster
+
+# Copy contents from builder
+RUN ls -lR /app/
+COPY --from=builder /app/static /app/static
+
+# Copy base files
+COPY banwebScrape.py /app/
+COPY course_gather /app/course_gather
+COPY manage.py /app/
+COPY requirements.txt /app/
+COPY run_tests /app/
```

Comment:
Doesn't look like we're using this in the build after this.

Reply:
Good point, should we add tests to the Docker flow? I don't see value in it currently, but if we did React tests I could see value.

Reply:
Personally, I would say that code testing shouldn't be part of the Dockerfile. Tests should be run before building the container. But that doesn't always happen so 🤷